resources/sts/4.11/sts_core_permission_policy.json

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
should the name be sts_installer_core_permission_policy?

Reply:
@gdbranco I was debating prefixing them all with `sts_installer` but I wasn't sure if I should. I could easily make the change though if you think it's better